docs/docs/50-configuration/01-server-configuration.md

```diff
@@ -2,36 +2,36 @@
-| `OBOT_SERVER_AUTH_OWNER_EMAILS` | A comma separated list of email addresses that will have the Owner role in Obot. |
-| `OBOT_SERVER_AUTH_ADMIN_EMAILS` | A comma separated list of email addresses that will have the Admin role in Obot. |
-| `OBOT_SERVER_OTEL_BASE_EXPORT_ENDPOINT` | The base export endpoint for OpenTelemetry |
-| `OBOT_SERVER_OTEL_SAMPLE_PROB` | The sampling probability for OpenTelemetry |
-| `OBOT_SERVER_OTEL_BEARER_TOKEN` | The bearer token for authentication with OpenTelemetry |
-| `OBOT_SERVER_AUDIT_LOGS_MODE` | Configures the storage backend for audit logs in Obot. Can be 'off', 'disk', or 's3' |
-| `OBOT_SERVER_AUDIT_LOGS_STORE_S3BUCKET` | The name of the S3 bucket to store audit logs in. |
-| `OBOT_SERVER_AUDIT_LOGS_STORE_S3ENDPOINT` | If config.OBOT_SERVER_AUDIT_LOGS_MODE is 's3' and you are not using AWS S3, this needs to be set to the S3 api endpoint of your provider. |
-| `OBOT_SERVER_AUDIT_LOGS_COMPRESS_FILE` | Controls whether or not to compress audit log files |
-| `OBOT_SERVER_AUDIT_LOGS_USE_PATH_STYLE` | Whether to use path style for S3 |
-| `OBOT_SERVER_MCPBASE_IMAGE` | Deploy MCP servers in the kubernetes cluster or using docker with this base image. |
-| `OBOT_SERVER_MCPRUNTIME_BACKEND` | The runtime backend to use for running MCP servers: docker, kubernetes, or local. Defaults to docker. |
-| `OBOT_SERVER_MCPCLUSTER_DOMAIN` | The cluster domain to use for MCP services. Defaults to cluster.local. Only matters if `OBOT_SERVER_MCPBASE_IMAGE` is set. |
-`OBOT_SERVER_DISALLOW_LOCALHOST_MCP` | disallow MCP servers that try to connect to localhost. Defaults to false. |
+| Environment Variable | Description | Default |
+|---------------------|-------------|---------|
+| `OPENAI_API_KEY` | The foundation of Obot is a large language model that supports function-calling. The default is OpenAI and specifying an OpenAI key here will ensure none of the users need to worry about specifying their own API key. | - |
+| `ANTHROPIC_API_KEY` | You can also provide an Anthropic API key in place of or in addition to an OpenAI API key. | - |
+| `GITHUB_AUTH_TOKEN` | Obot anObot makes heavy use of repositories hosted on GitHub. Care is taken to cache these and only re-check when necessary. However, rate-limiting can happen. Setting a read-only token here can alleviate many of these issues. No grants are required for either a 'classic' or 'fine-grained' token to access public repos (read-only). If you want to give the token access to private repos, you will need to give it `repo` (for a 'classic' token) or `contents (read-only)` and `metadata (read-only)`. |
+| `OBOT_SERVER_DSN` | Obot uses a database backend. By default, it will use a sqlite3 local database. This environment variable allows you to specify another database option. For example, you can use a postgres database with something like `OBOT_SERVER_DSN=postgres://user:password@host/database`. | - (an internal postgresql database is used in the container) |
```

Comment:
```suggestion
| `OBOT_SERVER_DSN` | Obot uses a database backend. By default, it will use a sqlite3 local database when running the plan Obot binary. The Obot container will use an internal PostgreSQL database (not recommended for production). This environment variable allows you to specify another database option. For example, you can use a postgres database with something like `OBOT_SERVER_DSN=postgres://user:password@host/database`. | - (an internal postgresql database is used in the container, but this is not recommended for production) |
```